resim/metrics/default_report_metrics.py

```diff
@@ -0,0 +1,729 @@
+    column is a status like so:
+
+           PASSED  FAIL_WARN  FAIL_BLOCK  ERROR  CANCELLED  UNKNOWN
+        0      22         10          18      0          0        0
+        1      23         10          17      0          0        0
+        2      23         10          17      0          0        0
+        3      22         10          18      0          0        0
+        4      21         10          19      0          0        0
+
+    Args:
+        batch_ids: The batches whose statuses we care about.
+        batch_to_jobs_map: All of the jobs per batch.
+
+    Returns:
+        A dataframe containing status counts where each row is a batch in order
+        and each column is a status.
+    """
+    status_counts = []
+    for batch_id in batch_ids:
+        status_counts.append(_count_job_statuses(batch_to_jobs_map[batch_id]))
+
+    status_counts = pd.DataFrame(
+        data=np.array(status_counts),
+        columns=["PASSED", "FAIL_WARN", "FAIL_BLOCK", "ERROR", "CANCELLED", "UNKNOWN"],
+        index=range(len(batch_ids)),
+    )
+    return status_counts
+
+
+def job_status_categories_metric(
+    writer: ResimMetricsWriter,
+    *,
+    batches: list[Batch],
+    batch_to_jobs_map: dict[str, list[Job]],
+    job_to_metrics_map: dict[str, list[JobMetric]],
+    batch_to_metrics_map: dict[str, list[BatchMetric]],
+    scalar_batch_metrics_map: dict[str, dict[str, MetricProto]],
+) -> None:
+    # pylint: disable=unused-argument
+    """Create a variety of job status metrics.
+
+    Includes a filled line plot of statuses alongside aggregate metrics on:
+     - Number of batches
+     - Number of jobs
+     - Number of passing batches
+     - Number of failing batches
+     - Failed jobs on most recent batch
+     - etc.
+
+    Args:
+        writer: A metrics writer to write these metrics to.
+        batches: All batches for this report.
+        batch_to_jobs_map: All jobs for each batch.
+        job_to_metrics_map: All job metrics for each job.
+        batch_to_metrics_map: All batch metrics for each batch.
+        scalar_batch_metrics_map: All scalar batch metrics for each batch as protos.
+    """
+    status_counts = _count_batch_statuses(
+        list(batch_to_jobs_map.keys()), batch_to_jobs_map
+    )
+
+    fig = px.area(
+        status_counts,
+        x=range(len(batch_to_jobs_map)),
+        y=["PASSED", "FAIL_WARN", "FAIL_BLOCK", "ERROR", "CANCELLED", "UNKNOWN"],
+    )
+    fig.update_layout(
+        template="plotly_dark",
+        plot_bgcolor="rgba(0, 0, 0, 0)",
+        paper_bgcolor="rgba(0, 0, 0, 0)",
+    )
+
+    (
+        writer.add_plotly_metric("Job Statuses Over Time")
+        .with_description("Job Statuses Over Sequential Batches")
+        .with_blocking(False)
+        .with_should_display(True)
+        .with_importance(mu.MetricImportance.HIGH_IMPORTANCE)
+        .with_status(mu.MetricStatus.PASSED_METRIC_STATUS)
```

Comment:
Not Applicable?

Reply:
I guess we haven't really determined what our guidance on statuses is... I would suggest that most of these report metrics be Not Applicable just because we can't make a determination ourselves?

Thoughts?

Reply:
I think putting it as not-applicable is fine for now? This page is mostly supposed to be informational so it won't be needed to gate things (i.e. we don't need programmatic handling of the status).

Reply:
I've changed them all to NOT_APPLICABLE.